lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -23,6 +20,7 @@ import {
   itineraryHasAccessibilityScores
 } from '../../../util/accessibility-routing'
 import { getActiveSearch, getFare } from '../../../util/state'
+import { getFormattedMode } from '../../../util/i18n'
```

Comment:
actually, I think this is only used by `modeStrings` on line 324, which is also not getting used. You may be able to take them both out!